packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -130,6 +131,11 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
     log.info('> Cancelling download. Closing payment channels, and then removing torrent');
     const torrent = this.torrents.find(t => t.infoHash === infoHash);
     if (torrent) {
+      // I am only allowed to close the channel on my turn.
+      // If I don't pause the torrent, then I will continue to make payments, meaning the call to 
+      // this.closeChannels would race the `makePayment` function, and one of them would fail with a
+      // "Not my turn" error.
+      torrent.pause();
```

Comment:
The [documentation for pausing](https://webtorrent.io/docs) says: Temporarily stop connecting to new peers. Note that this does not pause new incoming connections, nor does it pause the streams of existing connections or their wires.

Given this, is it true that the leecher would stop making payments?

Reply:
Yes, according to the docs it only makes it stop making connections, but in Web3Torrent it also sets the `torrent.pause` to `true` which, in here: https://github.com/statechannels/monorepo/compare/w3t-close-channel?expand=1#diff-a84153ed9f3ab2bd13c41bca43b584c7R404-R405
it makes the leecher stop making payments.
This has the effect that the leecher doesn't pollutes the payment channel with messages when the closing flow is running, and avoids *some* errors on that flow.